kernel/src/process.rs

```diff
@@ -488,9 +488,12 @@ impl<'a> Process<'a> {
 
                 HAVE_WORK.set(HAVE_WORK.get() + 1);
 
+                debug!("Loaded process: {}", process.package_name);
```

Comment:
Hah. Umm.. it'll `panic!` and say "debug never set up" 😊 . I'll add console wiring for the other platforms

I'm not sure I actually agree with your general point there. I think it makes more sense to have `debug!` compile to a no-op for release builds, but print in debug builds. I think having a few 'standard debugging' things (i.e. not high-volume prints, but useful info for traces) would be a good thing to have -- thinking to the future when someone reports an issue, you ask them to flash a debug kernel, there are probably some messages that are always useful to get a sense of the state of the world. Which apps are running certainly qualifies for that.